sdk/loadtestservice/azure-developer-loadtesting/pom.xml

```diff
@@ -76,4 +80,29 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Do we have reason to make a different configuration (instead of use that from parent pom)?

Reply:
I might be wrong but code coverage did not work until we put this configuration

Reply:
I am not sure why it is needed (I assume coverage does not need it), e.g.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/devcenter/azure-developer-devcenter/pom.xml
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2011172&view=codecoverage-tab

models likely empty already, implementation is actually counted in coverage (it only means user does not use it, it does not mean our code does not use it).

You can have another draft PR to test changes in pom and how that works out. This likely not affect the code.

Reply:
Are the default minimum line and branch coverage values supposed to be used? If we remove this section, then we would not be able to set those values

Reply:
@weidongxu-microsoft After removing the jacoco excludes, line and branch coverage reduces a lot. Slight reduction is expected but in this case it dropped a lot. We have included test cases for most APIs. Do you have any recommendations for the kind of test cases we should have to meet code coverage goals?

Reply:
I don't think absolute number of coverage matters too much. There are some lines that only cover failure case, or invalid input case. I don't think we really need to cover all that.

Anyway, the default is 40% line coverage and 30% branch coverage. As long as you almost have tests for all APIs, and the number (even a bit lower) is about OK as this, you are good.